bdtopo_v2/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/bdtopo_v2/InputDataFormattingTest.groovy

```diff
@@ -100,7 +101,7 @@ class InputDataFormattingTest {
         def table = h2GISDatabase.getTable(tableName)
         assertNotNull(table)
         assertEquals(10, table.columnCount)
-        assertEquals(20568, table.rowCount)
+        assertEquals(2231, table.rowCount)
```

Comment:
This count is related to the datainput ?

Reply:
yes, the input data has changed (smaller one with the new impervious tables)